src/deps/uws.zig

```diff
@@ -1040,7 +1040,215 @@ pub fn NewSocketHandler(comptime is_ssl: bool) type {
         }
     };
 }
+pub fn NewWrappedSocketHandler(comptime is_ssl: bool) type {
```

Comment:
Maybe instead of adding another wrapper here, we could do this on the inner NewSocketHandler struct?

Reply:
Yeah but we would possible touch more places than just socket.zig so I was just cautious, I can refactor so everyone using NewSocketHandler also uses the detachable socket, I think http.zig and probably WebSocket client also uses it